qt/python/mantidqt/project/plotsloader.py

```diff
@@ -124,7 +122,11 @@ def workspace_plot_func(self, workspace, axes, fig, creation_arg):
         func = function_dict[function_to_call]
         # Plotting is done via an Axes object unless a colorbar needs to be added
         if function_to_call in ["imshow", "pcolormesh"]:
-            func([workspace], fig)
+            if creation_arg['normalize_by_bin_width']:
+                is_norm = creation_arg.pop('normalize_by_bin_width')
```

Comment:
Just checking why you need to pop the normalise argument, why can't you just do 
`func([workspace], fig, normalize_by_bin_width=creation_arg['normalize_by_bin_width'])`